backend/src/graphql/types/type/FILED.gql

```diff
@@ -5,7 +5,7 @@ type FILED {
   submitter: User
 }
 
-# this list equals the strings of an array in file "webapp/constants/modals.js"
+"this list equals the strings of an array in file `webapp/constants/modals.js`"
```

Comment:
Sure about that? I found no documentation on the `"` char. As far as I understand `#` is used for simple inline comments, whereas `"""` is a multiline comment which is added to the documentation

Reply:
May be here we want it as documentation:
![image](https://github.com/user-attachments/assets/da0c0add-f36b-4f6b-8d38-6a76fd023b6c)